main.go

```diff
@@ -74,6 +77,10 @@ func Main() int {
 		}
 	}
 	log.Infof("grol %s - welcome!", cli.LongVersion)
+	memlimit := debug.SetMemoryLimit(-1)
```

Comment:
A comment about the -1 would be appreciated, it's a magical number

Reply:
it's in the  doc https://pkg.go.dev/runtime/debug#SetMemoryLimit but will add

> A negative input does not adjust the limit, and allows for retrieval of the currently set memory limit.

Reply:
I know it's in the doc, but it's a bit cryptic when you don't know that you call set with -1 to get the value.

Thanks for adding the comment